categories.php

```diff
@@ -0,0 +1,22 @@
+<?php
+header('Content-Type: application/json; charset=utf-8');
+header('Access-Control-Allow-Origin: *');
+
+require('vendor/autoload.php');
+
+use Furniture\Classes\DbConnector;
+use Furniture\Classes\CategoryHydrator;
+use Furniture\Classes\CategoryEntity;
+use Furniture\Services\ResponseService;
+
+$successMessage = "Successfully retrieved categories";
+
+try {
+    $db = DbConnector::connectToDb();
+    $categories = CategoryHydrator::fetchAllCategories($db);
+    echo json_encode(ResponseService::createResponse($successMessage, $categories));
```

Comment:
have you checked what the FE does if there are no categories? you havent checked if `$categories` actually contains data, so this could send back no data as a success